.ci/flake8_wrapper_docstrings.sh

```diff
@@ -7,7 +7,7 @@ set -e
 # D3XX - Quoting issues.
 # D401 - First line should be in imperative mood 
 # D403 - First word of the first line should be properly capitalized
-args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D401,D403"
+args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D403"
```

Comment:
Right, this check was improved in https://github.com/PyCQA/pydocstyle/pull/235 .